src/controllers/signMessage/signMessage.ts

```diff
@@ -105,22 +101,6 @@ export class SignMessageController extends EventEmitter {
         this.dapp = dapp
       }
       this.messageToSign = messageToSign
-      const network = this.#networks.networks.find(
-        (n: Network) => n.id === this.messageToSign?.networkId
-      )
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      messageHumanizer(
-        messageToSign,
-        this.#storage,
```

Comment:
After these changes, `storage` and `fetch` are no longer needed for the SignMessage controller. Remove them?